src/handlers/ens.ts

```diff
@@ -89,3 +91,32 @@ export const fetchRegistration = async (recipient: any) => {
     }
   }
 };
+/**
+ * Get USD and ETH cost of the registration of a given name during given years
```

Comment:
Would reading the `rentPrice` from the ENS controller be a better source of truth?

https://etherscan.io/address/0x283Af0B28c62C092C9727F1Ee09c02CA627EB7F5#readContract

Reply:
Oops, didn't see your above comment, disregard!